assets/training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -10,3 +10,4 @@ wget=={{latest-pypi-version}}
 applicationinsights=={{latest-pypi-version}}
 azureml-automl-core=={{latest-pypi-version}}
 azureml-telemetry=={{latest-pypi-version}}
+pycocotools==2.0.4
```

Comment:
This needs to move to the conda section. Build fails for this with pip. 

... or we need to upgrade to 2.0.6

Reply:
updated to 2.0.6 in pip dep